src/core/sys/posix/sys/stat.d

```diff
@@ -534,6 +534,83 @@ version( CRuntime_Glibc )
         else
             static assert(stat_t.sizeof == 128);
     }
+    else version (SPARC64)
+    {
+        private
+        {
+            alias __dev_t = ulong;
+            alias __ino_t = c_ulong;
+            alias __ino64_t = ulong;
+            alias __mode_t = uint;
+            alias __nlink_t = uint;
+            alias __uid_t = uint;
+            alias __gid_t = uint;
+            alias __off_t = c_long;
+            alias __off64_t = long;
+            alias __blksize_t = c_long;
+            alias __blkcnt_t = c_long;
+            alias __blkcnt64_t = long;
+            alias __timespec = timespec;
+            alias __time_t = time_t;
+        }
+        struct stat_t
+        {
+            __dev_t st_dev;
+            ushort __pad1;
+            __ino_t st_ino;
```

Comment:
This entire block is a copy pasto from `version (AArch64)`, I just diff'd the C headers and changed what was different.  Can remove, it is always false anyway.

Something should probably be done about those always false identifiers in sys/posix/config. But not now.